styles/components/modal/_core.scss

```diff
@@ -0,0 +1,82 @@
+// Modal structure, taken from ember-modal-dialog/ember-modal-structure
+.ember-modal-overlay {
+  height: 100vh;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 50;
```

Comment:
I know we don't have z-index vars in the lib yet but lets go ahead and add a `$adde-modal-z-index` var to the modals var file, that way we can update easier when we add z-index for vars realz